components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonConstants.java

```diff
@@ -83,6 +83,10 @@
 
     public static final String READ_ONLY_USER_CLAIM = "http://wso2.org/claims/identity/isReadOnlyUser";
 
+    public static final String PROVISIONED_SOURCE_CLAIM = "http://wso2.org/claims/identity/userSource";
+    public static final String SYNCED_ATTRIBUTE_EDITING_ENABLED = "JITProvisioning.EnableSyncedAttributeEditing";
```

Comment:
Is there any single place we keep JIT provisioning-related configs?

Reply:
JIT provisioning related configs are maintained here[1].

[1] https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2#L1266-L1269